src/macros.jl

```diff
@@ -396,7 +405,7 @@ function parse_one_operator_constraint(_error::Function, vectorized::Bool,
     parsecode, buildcall
 end
 
-function parse_one_operator_constraint(_error::Function, vectorized::Bool, sense::Val, lhs, rhs)
+function parse_one_operator_constraint(_error::Function, vectorized::Bool, sense::Sense, lhs, rhs)
```

Comment:
The [SetProg](https://github.com/blegat/SetProg.jl) extension uses sense which is `Val{:⊆}` so I would prefer to keep `Val` here. The method is more specific than the other one anyway since it has `lhs, rhs` instead of `args...`.